packages/cli/src/CredentialsOverwrites.ts

```diff
@@ -1,14 +1,17 @@
-import config from '@/config';
+import { Service } from 'typedi';
 import type { ICredentialDataDecryptedObject } from 'n8n-workflow';
-import { deepCopy, LoggerProxy as Logger, jsonParse, ICredentialTypes } from 'n8n-workflow';
+import { deepCopy, LoggerProxy as Logger, jsonParse } from 'n8n-workflow';
+import config from '@/config';
 import type { ICredentialsOverwrite } from '@/Interfaces';
+import { CredentialTypes } from '@/CredentialTypes';
 
-class CredentialsOverwritesClass {
+@Service()
+export class CredentialsOverwrites {
```

Comment:
For understanding: What's the difference between the overwrites that happen in `CredentialsOverwrites` vs. the overwrites that happen in the `FrontendService`? Why do we need both and for them to be separate?

Reply:
Skimming through them they look very similar and with no context or method documentation I do not find the difference clear. Running low on time so I'll leave this be for now.